packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -37,15 +40,44 @@ const GuidanceMessage = styled.strong`
   }
 `;
 
+const GuidanceMessage = styled.strong`
+  ${baseTextStyles}
+`;
+
+const StyledNoScript = styled.noscript`
+  strong {
+    ${baseTextStyles}
+    bottom: 0;
+  }
+`;
+
 const Guidance = ({ guidanceMessage, service }) => (
-  <GuidanceWrapper service={service}>
-    <GuidanceMessage aria-hidden="true">{guidanceMessage}</GuidanceMessage>
+  <GuidanceWrapper
+    service={service}
+    className="guidance-wrapper"
+    guidanceMessage={guidanceMessage}
+  >
+    {guidanceMessage && (
+      <GuidanceMessage className="guidance-message" aria-hidden="true">
+        {guidanceMessage}
+      </GuidanceMessage>
+    )}
+    <StyledNoScript>
+      <strong>
+        This video cannot play in your browser. Please enable Javascript or try
+        a different browser.
+      </strong>
+    </StyledNoScript>
   </GuidanceWrapper>
```

Comment:
@simonsinclair we are reusing most of the styles as `baseTextStyles` on line 29 but separating them to mean different things. I think if we were to build on adding more stylistic requirements on either of them it would be better if they don't use the same component

Reply:
> if we were to build on adding more stylistic requirements on either of them

Could you give an example of what you are envisaging here? I would also have expected us to reuse existing components where possible (at least until such a time as we _know_ they have to diverge)

Reply:
@jamesdonoh eg. if we decided to make font-specific changes or add some more ux requirement to this and not on the guidance message

Reply:
I think let's cross that bridge when we come to it, if we already have something that serves our current requirements.

Reply:
done